codegen/src/main/java/com/paypal/selion/elements/HtmlSeLionElement.java

```diff
@@ -53,6 +56,20 @@
     public String toString() {
         return String.format("%s, canHaveParent=%s",super.toString(), canHaveParent);
     }
+    
+    /**
+     * By providing the qualified name of a custom element we can register it to the element array.
+     * Custom elements are inserted before SeLion elements, if you use the same name it will overwrite the existing element.
+     * 
+     * @param element string of the qualified class
+     */
+    public static void registerElement(String element) {
+        List<HtmlSeLionElement> temp = new ArrayList<HtmlSeLionElement>(Arrays.asList(values));
+        
+        temp.add(0, new HtmlSeLionElement(HtmlElementUtils.getPackage(element), HtmlElementUtils.getClass(element), true, false));
+        
+        values = temp.toArray(new HtmlSeLionElement[temp.size()]);
```

Comment:
So the "overwrite" of an existing standard SeLion element value should at least emit a log message / warning here?  I assume this is not going to be a typical use case here with custom elements? Is there really a need to allow this use case since the order of the elements are matched is important and I can see this tripping up folks who are not careful.